nasa/earthdata-varinfo
# Trt 556 final schema clean up

## Description

This PR performs the final clean-up relating to the recommendations for cleaning up the `earthdata-varinfo` configuration file schema (recommendations 4 to 7 [here](https://wiki.earthdata.nasa.gov/display/SITC/earthdata-varinfo+configuration+schema+v2.0.0+proposal)). Specifically:

* `Grid_Mapping_Data` is removed (as this information wasn't directly used, and can be included as metadata attributes in `CFOverrides`.
* `ProductEpochs` is removed (as this information wasn't directly used, and can be included as a `units` metadata attribute in the more CF Convention compliant manner via `CFOverrides`).
* The schema now uses `PascalCase` for all attributes of the schema.

This is mostly small stuff updates, and I think it closes out the last of the proposed schema changes.

## Jira Issue ID

TRT-556

## Local Test Steps

* Pull this branch.
* Pip install all the local dependencies.
* Try and parse a local NetCDF-4 or DMR file. It should all work.

```
# Install dependencies:
$ pip install -r requirements.txt

# Install this version of the package (optional, if running from the root of the directory):
$ pip install -e .

# Parse a file (with/without a configuration file, this is now in a Python repl):
> from varinfo import VarInfoFromNetCDF4
>
> vi_instance = VarInfoFromNetCDF4('path/to/file.nc4', short_name='<collection short name>', config_file='/path/to/config.json')
```

## PR Acceptance Checklist
* ~~`VERSION` updated if publishing a release.~~ (This is a feature branch)
